manifests/a/Acronis/CyberProtectHomeOffice/26.3.1.39900/Acronis.CyberProtectHomeOffice.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Automatically updated by the winget bot at 2022/May/10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.3.1.39900
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: -silent_installation
+  SilentWithProgress: -silent_installation
+  Log: -log:"<LOGPATH>"
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.acronis.com/u/AcronisCyberProtectHomeOffice.exe
+  InstallerSha256: 9E60D8CE605EC870BA5AC26B97DD75B174EE33F5597A515142C25AE8ABE5D88D
+  Scope: machine
+  AppsAndFeaturesEntries:
+  - DisplayName: Acronis Cyber Protect Home Office
+    DisplayVersion: 26.1.39703
+    InstallerType: msi
+    ProductCode: '{E1FEA3C4-5790-465E-ADDD-6FF1E9FDCAB4}'

```

Comment:
```
  - DisplayName: Acronis Cyber Protect Home Office
    DisplayVersion: 26.1.39703
    InstallerType: msi
    ProductCode: '{E1FEA3C4-5790-465E-ADDD-6FF1E9FDCAB4}'
```
This probably shouldn't be here since WinGet ignores anything that has `SystemComponent` set to `1` in the registry so this wouldn't be used for ARP.